### PR TITLE
Basic OI Framework

### PR DESCRIPTION
**Pull request by N8-Des**
This should be all that was asked for the OI framework.

### CODE REVIEW

File: src/main/java/org/frc5687/deepspace/robot/utils/Gamepad.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
+package org.frc5687.switchbot.robot.utils;
+
```
Comment (BenBernardCIS) [inline src/main/java/org/frc5687/deepspace/robot/utils/Gamepad.java]: When you copy a file betwen projects, you have to correct the package name.

File: src/main/java/org/frc5687/deepspace/robot/OI.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import edu.wpi.first.wpilibj.Joystick;
```
Comment (BenBernardCIS) [inline src/main/java/org/frc5687/deepspace/robot/OI.java]: This file is missing a pacakge statement...

File: src/main/java/org/frc5687/deepspace/robot/OI.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import edu.wpi.first.wpilibj.buttons.Button;
+import edu.wpi.first.wpilibj.buttons.JoystickButton;
+
+public class OI(){
```
Comment (BenBernardCIS) [inline src/main/java/org/frc5687/deepspace/robot/OI.java]: The parens above are incorrect...

File: src/main/java/org/frc5687/deepspace/robot/OI.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    protected Gamepad _driverGamepad;
```
Comment (BenBernardCIS) [inline src/main/java/org/frc5687/deepspace/robot/OI.java]: You'll need to import the Gamepad class once you correct its package.

File: src/main/java/org/frc5687/deepspace/robot/utils/Gamepad.java (hunk 1 above)
Comment (BenBernardCIS) [reply src/main/java/org/frc5687/deepspace/robot/utils/Gamepad.java]: (replace switchbot with deepspace)

**Comment by BenBernardCIS** [comment]
Fixes #6

File: src/main/java/org/frc5687/deepspace/robot/utils/Gamepad.java (hunk 1 above)
Comment (N8-Des) [reply src/main/java/org/frc5687/deepspace/robot/utils/Gamepad.java]: Oh, I am a fool. 
My bad.

**Pull request merged**